### PR TITLE
updating license on develop

### PR DESCRIPTION
**Pull request by wjcunningham7**
- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have read the CONTRIBUTING document.

### CODE REVIEW

File: .github/workflows/boilerplate.yml
Comment (kessler-frost) [inline .github/workflows/boilerplate.yml:1]: Do we want to remove the this workflow completely?

**Review by kessler-frost — CHANGES_REQUESTED** [review]
Pretty much everything looks good, just one comment about the `boilerplate.yml` workflow. I don't think its a good idea to get rid of it, I have a suspicion that its removal was accidental 😶 .

**Comment by kessler-frost** [comment]
Also, the changelog needs to be updated.

**Comment by Andrew-S-Rosen** [comment]
Closes #1745.

Thank you so much!!! 🙏

**Review by kessler-frost — APPROVED** [review]

**Pull request merged**